src/pages/FAQ/FAQ.test.jsx

```diff
@@ -22,6 +22,6 @@ describe("FAQ", () => {
   test("renders FAQ heading", () => {
     const wrapper = setup();
     const headingDisplay = findByTestAttr(wrapper, "heading-display");
-    expect(headingDisplay.length).toBe(1);
+    expect(headingDisplay.length).toBe(3);
```

Comment:
:pick: Update all of these to use `exists()`.

Reply:
Addressed:  62ea76e426f45d0cc1f295135945947f89142b64